src/main/java/seedu/jimi/model/FilteredListManager.java

```diff
@@ -297,6 +345,26 @@ public boolean run(ReadOnlyTask task) {
         }
         
     }
+    
+    /**
+     * Qualifier to check if task/event is overdue w.r.t. current time.
+     * @author zexuan
+     *
+     */
+    private class OverdueQualifier implements Qualifier {
+
+        @Override
```

Comment:
also can add a boolean in the constructor, like an "enable" like that of the TaskQualifier